### DISCUSSION TITLE
Add sorting to category field

### DISCUSSION THREAD

**Discussion by leevigraham** [craftcms/cms#9113] [Ideas]
It would be great if categories could be sorted like other relationship fields. This would allow editors to choose a "Primary Category" for eyebrow headings by just reordering the categories.

It looks like this has been explicitly disabled: https://github.com/craftcms/cms/blob/develop/src/web/assets/cp/src/js/CategorySelectInput.js#L10

… ahhhh … 

I've just realized by looking at the templates some more that categories can be nested which is probably why sorting is disabled. I wonder if this could still be implemented using the structure sorting functionality and only allowing parents to be sorted… 

Tricky one.

**Comment by brandonkelly** [comment]
Will be possible in Craft 4 (see #1949).
